package.json

```diff
@@ -13,11 +13,12 @@
     "babel-plugin-transform-es2015-destructuring": "^6.6.5",
     "babel-preset-es2015": "^6.6.0",
     "browser-sync": "^2.12.10",
-    "browserify": "^13.0.1"
+    "browserify": "^13.0.1",
+    "remarkable": "^1.6.2"
   },
   "scripts": {
     "start": "scripts/server.js",
-    "deploy": "npm run babel && npm run browserify && git subtree push --prefix site origin gh-pages",
+    "deploy": "npm run babel && npm run browserify && node site/scripts/generate_readme.js && git subtree push --prefix site origin gh-pages",
```

Comment:
`generate_readme` should be `generateReadme`.

Also, should this be a standalone `npm` task? Why or why not?